nice_trim.py

```diff
@@ -41,7 +41,7 @@ def get_devpath_from_mountpoint(mountpoint, log):
 def get_trimmable(log):
     'helper function'
     result = {}
-    lsblk_out = check_output(['lsblk', '-POb'])
+    lsblk_out = check_output(['lsblk', '-Pbo', '+DISC-GRAN'])
```

Comment:
Hmm, I can understand your point to help the experience during code reading and hands-on. In that direction,  further tweak is required, since the default output column is changed in the even more recent lsblk version, util-linux 2.37.2 in my hand,  namely, `MOUNTPOINTS` is printed by default, instead of `MOUNTPOINT`.

```suggestion
    lsblk_out = check_output(['lsblk', '-Pbo', '+DISC-GRAN,MOUNTPOINT,RO'])
```